plugin-attribution/src/main/java/com/mapbox/maps/plugin/attribution/AttributionViewImpl.kt

```diff
@@ -71,7 +71,12 @@ class AttributionViewImpl @JvmOverloads constructor(
    * @param bottom Margin to the bottom in pixel
    */
   override fun setAttributionMargins(left: Int, top: Int, right: Int, bottom: Int) {
-    (layoutParams as FrameLayout.LayoutParams).setMargins(left, top, right, bottom)
+    (layoutParams as FrameLayout.LayoutParams).apply {
+      marginStart = left
```

Comment:
Just curious, why not use `leftMargin` and `rightMargin` to be consistent with others?

Reply:
in order to support for RTL configuration as well, we use `startMargin` and `endMargin`, rather than `leftMargin` and `rightMargin`. Android system will ignore `left` and `right` in rtl configuration.